pkg/migration/error.go

```diff
@@ -0,0 +1,5 @@
+package migration
+
+import "errors"
+
+var NotFound = errors.New("not found")
```

Comment:
Do you mean this PR would look just like this?

Reply:
/cc @kopiczko

Reply:
Nah, as `microerror.Rootf()` is still missing, this would mean diverge from current focus; I'll just fall back to `fmt.Errorf()` for now.

Reply:
It would work.

You can do:

```
microerror.Mask(NotFound)
```

or:

```
microerror.Mask(fmt.Errorf("%w: extra information", NotFund))
```

And `errors.Is(err, NotFound)` should work for both cases.